README.md

```diff
@@ -1,2 +1,63 @@
-# Telegraf-Output-Apache-Arrow-Flight
-External Telegraf plugin for writing time series to Apache Arrow Flight
+# Apache Arrow Flight External Output Plugin
+
+This plugin is a general purpose output plugin for the [Apache Arrow Flight protocol](https://arrow.apache.org/docs/format/Flight.html).
+
+## Arrow Schema
+
+The plugin currently supports inserting data points with the following schema:
+
+```go
+schema:
+  fields: 3
+    - tid: type=int32
+    - timestamp: type=timestamp[ms]
+    - value: type=float32
+```
+
+Support for arbitrary schemas is planned.
+
+## Setting up and running the plugin
+
+To build the binary and run the plugin:
+
+1. Install the latest version of [Go](https://go.dev/doc/install).
+2. Build the binary:
+    * Windows: `go build -o binary/flight.exe cmd/main.go`
+    * Linux/macOS: `go build -o binary/flight cmd/main.go`
+3. Download the latest version of [telegraf](https://portal.influxdata.com/downloads/) for your platform. (To see which platform is needed, run the following command: `go env GOOS GOARCH`)
+4. Extract the telegraf executable and configuration to the repository folder.
```

Comment:
Maybe write "... Telegraf binaries ..." to match the language on the InfluxData website. This would also make it more clear what format to download Telegraf in, e.g., I assume the Docker Image is not supported when using these instructions.

Reply:
Fixed.